src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -132,6 +145,16 @@ public void deleteRoles(final Node node) throws RepositoryException {
                 rdfTypes.addAll(x.getTypes());
             });
 
+        // If we get the default Authorization and it contains acl:accessTo properties,
```

Comment:
Thanks, @acoburn, that makes sense... however, the block itself and associated comment did not shine much light in that direction for me. Would you please update the comment or pull this into its own private method to make the logic and intent more clear?